diff --git a/web/.htaccess b/web/.htaccess
index 83cb71a..f890739 100644
--- a/web/.htaccess
+++ b/web/.htaccess
@@ -69,6 +69,10 @@ AddEncoding gzip svgz
 <IfModule mod_rewrite.c>
   RewriteEngine on
 
+  RewriteCond %{HTTP_HOST} ^(www\.)?screening.mhanational\.org$ [NC]
+  RewriteCond %{HTTPS} !=on
+  RewriteRule ^ https://%{HTTP_HOST}%{REQUEST_URI} [L,NE,R=301]
+
   # Set "protossl" to "s" if we were accessed via https://.  This is used later
   # if you enable "www." stripping or enforcement, in order to ensure that
   # you don't bounce between http and https.
